packages/server/tests/acceptance/estimateGasPrecompile.spec.ts

```diff
@@ -710,45 +1130,181 @@ describe('EstimatePrecompileContract tests', function () {
     isWithinDeviation(gasResult.gasUsed, estimateGasResponse, lowerPercentBound, upperPercentBound);
   });
 
+  //EGP-043 - The test is excluded due to an issue in mirror node
+  it.skip('should call estimateGas with createFungible token function', async function () {
+    let estimateContractTokenCreate = new ethers.Contract(
+      prefix + EstimatePrecompileContractAddress,
+      EstimatePrecompileContractJson.abi,
+      accounts[0].wallet,
+    );
+
+    const gasOptions = await Utils.gasOptions(requestId, 15_000_000);
+    let gasLimit = gasOptions.gasLimit;
+    let gasPrice = gasOptions.gasPrice;
+
+    const tx = await estimateContractTokenCreate.createFungibleTokenPublic.populateTransaction(
+      accounts[0].wallet.address,
+      {},
+    );
+
+    console.log('Gas Limit is ' + tx.gasLimit);
```

Comment:
We have several "console.log" function calls. Do we need them ?

Reply:
those console.logs are in a test that is currently skipped due to an issue in mirror node, so will be updated and reworked and the unneeded lines will be removed